src/overlays/actors/ovl_En_Kakasi/z_en_kakasi.c

```diff
@@ -46,80 +77,1172 @@ const ActorInit En_Kakasi_InitVars = {
+    {0.0f, 55.0f, 60.0f,},
+    {0.0f, 55.0f, 60.0f,},
+    {-80.0f, 35.0f, 15.0f,},
+    {80.0f, 45.0f, 15.0f,},
+    {-10.0f, 35.0f, 65.0f,},
+    {0.0f, 110.0f, 180.0f,},
+    {0.0f, 60.0f, 70.0f,},
+    {0.0f, 60.0f, 70.0f,},
+    {-80.0f, 50.0f, 40.0f,},
+    {0.0f, -10.0f, 50.0f,},
+    {80.0f, 50.0f, 40.0f,},
+    {0.0f, 120.0f, 40.0f,},
+    {0.0f, 28.0f, -50.0f,},
+    {0.0f, 60.0f, 50.0f,},
+    {0.0f, 80.0f, 160.0f,},
+};
+
+Vec3f D_80971EEC[] = {
+    {12.0f, 1.0f, 2.0f,},
+    {12.0f, 1.0f, 2.0f,},
+    {2.0f, -6.0f, 0.0f,},
+    {12.0f, -6.0f, -10.0f,},
+    {-88.0f, 14.0, -10.0f,},
+    {0.0f, 0.0f, 0.0f,},
+    {0.0f, 0.0f, 0.0f,},
+    {0.0f, 0.0f, 0.0f,},
+    {0.0f, -10.0f, 0.0f,},
+    {0.0f, -10.0f, 0.0f,},
+    {0.0f, -10.0f, 0.0f,},
+    {0.0f, -10.0f, 0.0f,},
+    {0.0f, -10.0f, 0.0f,},
+    {0.0f, 0.0f, 0.0f,},
+    {0.0f, 0.0f, 0.0f,},
+};
+
+Vec3f D_80971FA0[] = {
+    {-50.0f, 160.0f, -40.0f,},
+    {-50.0f, 160.0f, -40.0f,},
+    {-20.0f, 80.0f, 60.0f,},
+    {57.0f, 60.0f, 120.0f,},
+    {27.0f, 80.0f, 60.0f,},
+    {-3.0f, 50.0f, 70.0f,},
+};
+
+Vec3f D_80971FE8[] = {
+    {0.0f, 0.0f, 0.0f,},
+    {0.0f, 0.0f, 0.0f,},
+    {-10.0f, -5.0f, 0.0f,},
+    {10.0f, -10.0f, 7.0f,},
+    {20.0f, 5.0f, 2.0f,},
+    {10.0f, 5.0f, 2.0f,},
+};
+
+// both use in digging away function
+Vec3f D_80972030 = { -3.0f, 50.0f, 90.0f, };
+Vec3f D_8097203C = { 10.0f, -15.0f, 2.0f, };
+
+typedef enum {
+    ENKAKASI_ANIME_ARMS_CROSSED_ROCKING = 0, // arms crossed and rocking back and forth
+    ENKAKASI_ANIME_SIDEWAYS_SHAKING = 1,
+    ENKAKASI_ANIME_HOPPING_REGULAR = 2,    // bounce bounce bounce (repeat)
+    ENKAKASI_ANIME_SPIN_REACH_OFFER = 3,   // spin and then reaching toward player
+    ENKAKASI_ANIME_TWIRL = 4,              // slower cinimatic twirl
+    ENKAKASI_ANIME_ARMS_CROSSED_STILL = 5, // arms crossed but still, also some noise sfx
+    ENKAKASI_ANIME_WAVE = 6,               // "wave" short sideways shake, stops early, partial? unused?
+    ENKAKASI_ANIME_SLOWROLL = 7,           // partial bounch, ends looking left, OFFER anim takes over
+    ENKAKASI_ANIME_IDLE = 8,               // slow stretching wiggle, ends in regular position
+
+} EnKakasi_Animations;
+
+extern AnimationHeader D_06007444;
+extern AnimationHeader D_0600686C;
+extern AnimationHeader D_060081A4;
+extern AnimationHeader D_06007B90;
+extern AnimationHeader D_060071EC;
+extern AnimationHeader D_06007444;
+extern AnimationHeader D_0600686C;
+extern AnimationHeader D_060081A4;
+extern AnimationHeader D_06000214;
+
+// static AnimationHeaderCommon* kakasiAnimations[] = {
+static AnimationHeader* kakasiAnimations[] = {
+    &D_06007444, &D_0600686C, &D_060081A4, &D_06007B90, &D_060071EC, &D_06007444, &D_0600686C, &D_060081A4, &D_06000214,
+};
+
+// animation mode, passed to SkelAnime_ChangeAnim
+u8 D_8097206C[] = {
+    0x00, 0x00, 0x00, 0x02, 0x02, 0x02, 0x02, 0x02, 0x02, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+};
```

Comment:
```suggestion
// animation mode, passed to SkelAnime_ChangeAnim
u8 D_8097206C[] = {
    0x00, 0x00, 0x00, 0x02, 0x02, 0x02, 0x02, 0x02, 0x02,
};
```
Because there are 9 animations, the rest of this would be end-of-data padding. It's worth double checking whether you need an extra pad variable